workorders.lic

```diff
@@ -246,6 +247,7 @@ class WorkOrders
     stow_hands
 
     quantity.times do |count|
+      dispose("#{info['stock-name']} #{material_noun[material_volume+recipe['volume']]}") if (count > 0 && spare_stock)
```

Comment:
Thank you for renaming `material_[noun|volume]`, it makes this clearer.

Could you extract `#{material_noun[material_volume+recipe['volume']]}` into a well-named variable to help provide some context?

Reply:
Its hard to do that because of the loop its in

Reply:
Its not needed anymore.